java.sourceui/apichanges.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Search for CHANGEME in this document when copying and using it: -->
 <!--
-
```

Comment:
Why are you deleting these lines? You don't have to change apichanges.xml at all.